frontend/src/features/search/BigSearchBox.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import classNames from 'classnames'
+
+import Icon from '../../chrome/Icon'
+
+interface BigSearchBoxProps {
+  onSubmit: (q: string) => void
+  value: string | undefined
+  className?: string
```

Comment:
move `onSubmit` to the last field (function props after data props). change `value: string | undefined` -> `value?: string`